Task2/index.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Login Form Using local storage js-part3</title>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link
+      rel="stylesheet"
+      href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css"
+    />
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/js/bootstrap.min.js"></script>
+  </head>
+  <body>
+    <div class="container">
+      <h2>Login Form</h2>
+      <form action="#">
+        <div class="form-group">
+          <label for="email">Email:</label>
+          <input
+            type="email"
+            class="form-control"
+            id="email"
+            placeholder="Enter email"
+            name="email"
+          />
+        </div>
+        <div class="form-group">
+          <label for="pwd">Password:</label>
+          <input
+            type="password"
+            class="form-control"
+            id="psw"
+            placeholder="Enter password"
+            name="pwd"
+          />
+        </div>
+
+        <button
+          type="button"
+          onclick="saveData()"
+          id="save_btn"
+          class="btn btn-default"
+        >
+          Submit
+        </button>
+      </form>
+    </div>
+  </body>
+  <script>
+    let name = localStorage.getItem("name") ? localStorage.getItem("name") : "";
+    console.log(name);
+    if (name != "") {
+      alert("Please visit profile");
+      window.location.href = "dashboard.html";
+    }
+    function saveData() {
+      let email, psw;
+      email = document.getElementById("email").value;
+
+      psw = document.getElementById("psw").value;
+
+      let user_records = new Array();
+      user_records = JSON.parse(localStorage.getItem("users"))
+        ? JSON.parse(localStorage.getItem("users"))
+        : [];
+      if (
+        user_records.some((v) => {
+          return v.email == email && v.psw == psw;
+        })
+      ) {
+        alert("Login Pass");
+        let current_user = user_records.filter((v) => {
```

Comment:
Can be refactored: 

`let current_user = user_records.filter((v) => v.email == email && v.psw == psw)[0];`

@Poorvi-3pillar